docker-compose.yml

```diff
@@ -17,15 +17,12 @@ services:
       - db
       - infobase
     volumes:
-      # Persistent volume mount for installed git submodules
-      - ol-vendor:/openlibrary/vendor
       # Persistent volume mount for generated css and js
```

Comment:
These too! All of 'em :)

Reply:
We should migrate out `ol-build` and `ol-nodemodules` while keeping `solr-data` and `solr-updater-data`?

volumes:
  solr-data:
  solr-updater-data:
  ~ol-build:~
  ~ol-nodemodules:~

Reply:
Yep! I.e. all the code-related volumes are moved. The data-persistancy volumes should stay there.